src/qrCard/QrCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import cc from 'classcat'
+
+import Card from '_utils/card'
+import ItemInfo from 'itemInfo'
+import TextSubHeaderStrong from 'typography/subHeaderStrong'
+
+export interface QrCardProps {
+  readonly ariaLabel?: string
+  readonly className?: Classcat.Class
+  readonly imageUrl: string
+  readonly itemMainTitle?: string
+  readonly itemMainInfo?: string
+  readonly title: string
+}
+
+const QrCard = ({
+  ariaLabel,
+  className,
+  itemMainTitle,
+  imageUrl,
+  itemMainInfo,
+  title,
+}: QrCardProps) => (
+  <Card className={cc(['kirk-qrCard', className])}>
+    <TextSubHeaderStrong>{title}</TextSubHeaderStrong>
```

Comment:
I think you could have used a `<SubHeader>`, it would add the proper paddings.

Reply:
and a11y semantic.

But you might have a double padding issue since the top internal padding of the Card is not normalized yet (go for it if you need, I will normalize that latter, I need to do it for the TripCard anyway).